tests/test_atom_client.py

```diff
@@ -0,0 +1,335 @@
+# -*- coding: UTF-8 -*-
+import os
+import pytest
+import sys
+import vcr
+
+here = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.insert(0, here)
+
+from agentarchives.atom.client  import AtomClient, AtomError, CommunicationError
+
+THIS_DIR = os.path.dirname(os.path.abspath(__file__))
+AUTH = {
+    'host': '127.0.0.1',
+    'api_key': '68405800c6612599'
+}
+
+
+@vcr.use_cassette(os.path.join(THIS_DIR, 'fixtures/atom', 'test_levels_of_description.yaml'))
+def test_levels_of_description():
+    client = AtomClient(**AUTH)
+    levels = client.get_levels_of_description()
+    assert levels == [u'Collection', u'File', u'Fonds', u'Item', u'Part', u'Series', u'Subfonds', u'Subseries'] 
+
+
+@vcr.use_cassette(os.path.join(THIS_DIR, 'fixtures/atom', 'test_listing_collections.yaml'))
+def test_listing_collections():
+    client = AtomClient(**AUTH)
+    collections = client.find_collections()
+    assert len(collections) == 2
+    assert collections[0]['title'] == 'Top Fonds'
+    assert collections[0]['type'] == 'resource'
+
+
+@vcr.use_cassette(os.path.join(THIS_DIR, 'fixtures/atom', 'test_collection_list.yaml'))
+def test_collection_list():
+    client = AtomClient(**AUTH)
+    collection_ids = client.collection_list('test-fonds')
+    assert len(collection_ids) == 2
+    assert collection_ids[0] == 'test-subfonds'
+
+
+@vcr.use_cassette(os.path.join(THIS_DIR, 'fixtures/atom', 'test_note.yaml'))
+def test_rendering_record_containing_a_note():
+    client = AtomClient(**AUTH)
+    collections = client.find_collections()
+    print str(collections)
```

Comment:
Tests don't generally print things, since the test client will show where the error was. Also, agentarchives is python 2/3! So please use Python 3 style prints and use `from__future__ import print_function`